### ISSUE TITLE
Duplicate Wiki create during clone process

### ISSUE THREAD

**Issue by SPB21** [OfficeDev/microsoft-teams-apps-requestateam#94]
Hi @alexc-MSFT we have deployed the Request a Team app and have noticed that duplicate wiki's are being created when we clone an existing team even though the source channel has a single wiki.  The first wiki tab errors out with 'We're having trouble with showing your Wiki right now. Please try again later.' when you click on it, the second wiki tab seems to configure when you click on it. 
Thanks

**Comment by aravindmunna** [comment]
@SPB21 Yes i too even got the same issue. Reported the same to @alexc-MSFT  He is looking into the issue.
